oioioi/mp/templates/mp/registration-notice.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
```

Comment:
I know PA and OI copied this notice, but third time in a single codebase seems to indicate a structure issue. It might be fine for now, but an issue should be raised at least.